clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -233,41 +232,41 @@ default DeleteTopicsResult deleteTopics(Collection<String> topics) {
      * @return The DeleteTopicsResult.
      */
     DeleteTopicsResult deleteTopics(Collection<String> topics, DeleteTopicsOptions options);
-    
+
```

Comment:
Would it make sense to provide a default implementation to translate to `TopicCollection`?

Reply:
So would this default implementation take strings or IDs?

Reply:
Ah... I'm not sure. We could do that.